pybossa/jobs.py

```diff
@@ -1445,6 +1472,12 @@ def write_user(writer, user):
     def add_headers(writer):
         writer.writerow(exportable_attributes)
 
+    if isinstance(email_addr, list):
+        recipients = email_addr
+        current_app.logger.info(u'Scheduling export_all_users job send to admins')
```

Comment:
logging can be outside of if else?

Reply:
this function is also reused when user requests user report from GIGwork UI. I wanted to make the logger msg more clear about either it's from weekly job or from users' request, so i make the logger msg inside IF.

Reply:
seems little hacky to determine its weekly or user report using variable type here. Does it make sense to set logs under caller func instead?